<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_window</class>
 <widget class="QDialog" name="login_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>282</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="min_btn">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#min_btn{
border-image: url(:/img/min.png);
background-color: transparent;
}
QPushButton#min_btn::hover{
background-image: url(:/img/min_hover.png);
background-color: transparent;
}

QPushButton#min_btn::pressed{
background-image: url(:/img/min_pressed.png);
background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_btn">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-image: url(:/img/close.png);
background-color: transparent;
}
QPushButton::hover{
background-image: url(:/img/close_hover.png);
background-color: transparent;
}

QPushButton::pressed{
background-image: url(:/img/close_pressed.png);
background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>67</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>67</width>
           <height>67</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/img/AppID8.png) stretch;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLineEdit" name="ip_edit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="accessibleDescription">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>IP:(localhost default)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="port_edit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>Port:12345</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="name_edit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Nick Name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="login_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#login_btn{
border-width:5px;
border-image: url(:/img/pushbutton.png) 5 5 5 5;
}

QPushButton#login_btn::hover{
border-width:5px;
border-image: url(:/img/pushbutton_hover.png) 5 5 5 5;
}

QPushButton#login_btn::pressed{
border-width:5px;
border-image: url(:/img/pushbutton_pressed.png) 5 5 5 5;
}</string>
       </property>
       <property name="text">
        <string>log in</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../img/img.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>enterRoom()</slot>
 </slots>
</ui>
